pallets/poa/Cargo.toml

```diff
@@ -28,6 +32,9 @@ pallet-session = { git = "https://github.com/paritytech/substrate", default-feat
 sp-io = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
 sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
 sp-std = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
+sp-staking = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
```

Comment:
Did we already decide to use the staking from upstream?

Reply:
To import `SessionIndex` is defined as part of `sp-staking`

![image](https://user-images.githubusercontent.com/82595429/178387005-d39df582-35ca-4eb8-9771-54ab34aa354c.png)